modules/apps/data-engine/data-engine-rest-test/src/testIntegration/java/com/liferay/data/engine/rest/resource/v2_0/test/DataDefinitionResourceTest.java

```diff
@@ -350,7 +350,7 @@ public void testPostDataDefinitionByContentType() throws Exception {
 
 		// MustSetFields
 
-		_testDataDefinitionContentType.setAllowEmptyDataDefinition(false);
+		DataDefinitionTestUtil.setAllowEmptyDataDefinition(false);
```

Comment:
Hey @RenanUmV, you do not need to move this method to this util class. Use reflection to change the field `_allowEmptyDataDefinition` from `TestDataDefinitionContentType` class. Thx